lib/game.py

```diff
@@ -55,6 +70,15 @@ def draw_grass(self):
                         row * SIZE_PER_CELL, col * SIZE_PER_CELL, SIZE_PER_CELL, SIZE_PER_CELL)
                     pygame.draw.rect(self.screen, GRASS_COLOR, grass_blk)
 
+    # TODO: check for if snake hits the boundaries
+    def draw_elements(self):
```

Comment:
It looks like draw_elements get called every cycle, do we need to draw the background everytime? Or do we just need to draw the snake?